os/mm/Kconfig

```diff
@@ -166,3 +166,13 @@ config MM_SHM
 	---help---
 		Build in support for the shared memory interfaces shmget(), shmat(),
 		shmctl(), and shmdt().
+config DEBUG_FREE
```

Comment:
How about add this as a lower config of mm?
```
config DEBUG_MM
if DEBUG_MM
config DEBUG_MM_FREE
endif
```

Reply:
@sunghan-chang , I will fix this and submit the change